examples/webgl_interactive_cubes_gpu.html

```diff
@@ -69,7 +69,7 @@
 				scene.background = new THREE.Color( 0xffffff );
 
 				pickingScene = new THREE.Scene();
-				pickingTexture = new THREE.WebGLRenderTarget( 1, 1 );
+				pickingTexture = new THREE.WebGLRenderTarget( 1, 1, { colorSpace: THREE.SRGBColorSpace } );
```

Comment:
@donmccurdy Instead if this line, the issue can also be fixed by updating line 139 to:
```js
applyVertexColors( geometry, color.setHex( i, THREE.LinearSRGBColorSpace ) );
```
What is your preference here?